dubbo-common/src/main/java/org/apache/dubbo/config/ProtocolConfig.java

```diff
@@ -236,6 +238,9 @@ public class ProtocolConfig extends AbstractConfig {
      */
     private String jsonCheckLevel;
 
+    @Nested
+    private TripleConfig triple;
```

Comment:
triple = new TripleConfig();
Initial value check can be simplified

Reply:
Okay, I'll fix it.

Reply:
![image](https://github.com/apache/dubbo/assets/18413695/3299de2b-222e-4272-97fa-7ef4151d8ce4)

![image](https://github.com/apache/dubbo/assets/18413695/5a999656-ce1c-4340-ac10-c6f51966f35a)

@oxsean Maybe I should just put it in checkDefault and follow the old rules, do you think so?

Reply:
@finefuture make sense,  initialize the default values here can avoid the null check during in builder.
We can add a checkDefault in TripleConfig add call it in ProtocolConfig

Reply:
Okay，I will add it.